crawl-ref/source/dat/des/portals/bazaar.des

```diff
@@ -1548,6 +1548,65 @@ xT.xxxxxxxxxxx.Tx
 xxxxxxxxxxxxxxxxx
 ENDMAP
 
+# 3 to 4 shops, with an average of 3.5
+NAME:       bazaar_kenran_forest_clearing
+TAGS:       no_rotate no_vmirror
+ORIENT:     encompass
+WEIGHT:     5
+KITEM:      h = human skeleton
```

Comment:
Right now the placement of this skeleton is fixed: what about putting a few Hs scattered around the map, with `NSUBST: H = 1=h / *=t.` to make one of them the skeleton and replace the rest with trees or floor randomly.